### PR TITLE
Use 2captcha service to automatically verified the captchas

### PR DESCRIPTION
**Pull request by Scalpweb**
Implementation of 2Captcha service
Need to fill your API KEY into the config file before testing

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,8 +23,13 @@ var email_domain = configFile.emailDomain;
 var lat = configFile.latitude;
 var lon = configFile.longitude;
 var country = configFile.country;
+var useAutoCatcha = configfile.useAutoCatcha;
```
Comment (joostsijm) [inline index.js:26]: Forgot to capitalize the f in "file".

**Review by FrostTheFox — CHANGES_REQUESTED** [review]
Change to not use autocaptcha by default, and fix the f as mentioned by joostsijm. little confused about the "catcha", but it's used consistently so i guess its ok.

**Comment by FrostTheFox** [comment]
Actually, once I get feedback from my testers I might just merge in and fix myself.

**Review by FrostTheFox — APPROVED** [review]

**Comment by Scalpweb** [comment]
Oh well... I commited the changes anyway.

**Pull request merged**